django/db/models/window.py

```diff
@@ -0,0 +1,186 @@
+from django.db import connection
```

Comment:
Maybe `expressions.py` should be converted into a package and this moved into there?

Reply:
Move the new expressions to this file, and reorganize later?